assets/scss/custom.scss

```diff
@@ -340,6 +340,7 @@ footer {
   }
 
   table {
+    display: table;
```

Comment:
somewhat surprised this is necessary.

Reply:
I've traced this back to https://github.com/twbs/bootstrap/blob/e5643aaa89eb67327a5b4abe7db976f0ea276b70/scss/_tables.scss#L173 causing tables to have `display: block`. Docsy makes all tables responsive: https://github.com/google/docsy/blob/d9bf98b876d74772b850ef910c6f59b898f61ccf/assets/scss/_content.scss#L41

Bootstrap 5.0.0+ has removed the `display: block` bit: https://github.com/twbs/bootstrap/pull/30482. However, Docsy is currently stuck on Bootstrap 4.6.2: https://github.com/google/docsy/issues/470.

https://github.com/google/docsy/commit/1e7e32934f520cfe1baaecfe6576db42addc89d1 did indeed cause this, as when adding a `>` back in front of `table` again, the `display: table` is no longer needed. I'm not sure if that's unintended behaviour for the PR, so I've left a comment.

Including `display: table` here seems to be an ample workaround for right now though. I'll leave a comment mentioning why it's needed.

Reply:
sounds good. I don't see a comment yet though?

Reply:
Oops. High aspirations. Have left both comments now, the one here in f261919feebad7e591ed08770c354ef2a1f3305c.

Reply:
Great in-depth analysis and explanation. Thanks for letting us know on the Docsy side as well. As you write, hopefully this will be resolved once Docsy upgrades to Bootstrap 5.